src/main/java/sirius/biz/mongo/SortField.java

```diff
@@ -60,6 +60,10 @@ public SortField(MongoEntity owner) {
      * @return the normalized text in lower-case
      */
     public static String normalizeText(String text) {
+        if (Strings.isEmpty(text)) {
+            return null;
```

Comment:
```suggestion
            return text;
```

i would suggest to just return the input if its null/empty and not change an empty string to null

Reply:
The underlying sort field would treat both the same so I commonly like normalizers to convert both to null to have a more predictable output. However, it can be changed here to make the PR less impactful.